packages/lib/modules/tokens/TokenInput/TokenInput.tsx

```diff
@@ -262,6 +262,7 @@ export const TokenInput = forwardRef(
                 boxShadow="none"
                 fontSize="3xl"
                 fontWeight="medium"
+                isDisabled={!token}
```

Comment:
This could have an impact in my Api Tokens refactor PR. I will take a look when I can.